src/NLog/LayoutRenderers/Wrappers/ReplaceNewLinesLayoutRendererWrapper.cs

```diff
@@ -75,7 +75,7 @@ protected override void RenderInnerAndTransform(LogEventInfo logEvent, StringBui
                 {
                     string str = builder.ToString(orgLength, builder.Length - orgLength);
                     str = str.Replace(newLine, Replacement);
-                    if (newLine != "\n" && !(Replacement?.IndexOf('\n') >= 0) && str.IndexOf('\n') >= 0)
+                    if (newLine != "\n" && Replacement?.IndexOf('\n') < 0 && str.IndexOf('\n') >= 0)
```

Comment:
@snakefoot could you please double check this change? Thanks!

Reply:
Hehe I suck at null-conditional operators :)

Reply:
yeah, doubting if its the same, maybe have to use Replacement!=null explicit,